packages/edit-site/src/components/sidebar-navigation-screen-navigation-item/index.js

```diff
@@ -51,6 +51,11 @@ export default function SidebarNavigationScreenNavigationItem() {
 					icon={ pencil }
 				/>
 			}
+			description={
+				postType === 'page'
+					? __( 'This is a static page.' )
```

Comment:
Let's do something that helps explain what pages are, if possible. For example, from https://make.wordpress.org/support/user-manual/posts-vs-pages/#about-pages

"Pages are static and are not listed by date. Pages do not use tags or categories."

"Posts are entries listed in reverse chronological order on the site homepage or on the posts page."